doc/sphinx/source/recipes/recipe_gier20bg.rst

```diff
@@ -194,6 +196,8 @@ Example plots
    of interannual variability.
 
 .. _fig_gier20bg_5:
+..
+  gallery
```

Comment:
If there are two `gallery` comments, it will simply show both plots in the gallery, right?